### DISCUSSION TITLE
2021-07-14 Weekly Meeting 12 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#60] [Weekly Meetings]
Weekly Meeting 12 is happening on Wednesday 2021-07-14 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- Website effort touchbase #49 
- GLTF extensions working group touch base #64 
- Fiscal Development Group touch base #57 

### Add-ons
- #59 
- #63 
- Jin would like to share some progress on incentive structures and discord tools for better online coordination

**Comment by mrmetaverse** [comment]
When we spend time together, we learn from each other. A critical part of our community development relies on our ability to communicate and document our findings. If you look at the agenda above, it's easier to see how our weekly meetings could soon lack enough time to discuss all of our working groups efforts. We would like to propose that a small team form to help plan the process for demo days where we will share our recent findings with each other. 

We created [ticket #59](https://github.com/omigroup/OMI/issues/59) to kick off the planning. We have a number of questions before we can press play. For example:
- how often should / can we meet? 
- how should we handle signing up to speak? 
- what is considered in and out of scope? 
- who can participate? 
- where do we document our findings for others? 
- is this actually going to be useful for our community? 
- how do we keep it lightweight and simple? This should be about sharing knowledge or recent lessons learned related to Interoperability, Open Source Metaverse, and OMI's work. _These are separate from lightning talks which can be about anything._ 

This conversation could take up an entire meeting, so I propose that we form a small team and plan the process before setting a date.

**Comment by mrmetaverse** [comment]
Today Robert and I created Issue #63 to create an "OMI weekly meeting discord bot". 

### Acceptance Criteria:
Takes attendance
Track voting

_Additional criteria can be tracked on the ticket [here](https://github.com/omigroup/OMI/issues/63)_

**Comment by mrmetaverse** [comment]
+ Jin would like to share some progress on incentive structures and discord tools for better online coordination

**Comment by mrmetaverse** [comment]
Meeting notes started on [hackmd](https://hackmd.io/@mrmetaverse/omi-071421)
